recipes/sigmut/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+./sigprofiler -h
```

Comment:
Please remove this line

Reply:
Hi @dpryan79, thanks a lot for your help ! I will modify the files and make the PR again.

Reply:
Please just edit your PR in place rather than making a new one :)

Reply:
Totally agree and sorry for this inconvenience. Actually I've created a new PR because I've committed from a different fork (from workplace), but I've forgotten to close the previous one.